vega_sim/scenario/ideal_market_maker_v2/scenario.py

```diff
@@ -145,6 +151,7 @@ def set_up_background_market(
             tag=str(tag),
             buy_intensity=self.buy_intensity,
             sell_intensity=self.sell_intensity,
+            random_state=random_state,
```

Comment:
Should we also add 'random state' in MarketOrderTrader agent when posting MOs to fix randomness?

Reply:
I think that's what this is doing here no? Or am I missing using it somewhere?

Reply:
Ah, I confused it with another agent, my bad. Looks fine to me now.